### DISCUSSION TITLE
srsRAN with 5G NR-U

### DISCUSSION THREAD

**Discussion by Codeternel** [srsran/srsRAN_Project#734] [General Help]
Hello,

I'm writing because I'm interested in deploying a non-private 5G network to highlight a use case in the industrial context. To do so, I will use the unlicensed spectrum of the 5G NR, the so-called NR-U.

I was wondering if there are any perspective for using or supporting the use of 5G NR-U in this project in the near future? If so, would you have any details on this regarding, for example, a deployment date?

Thank you in advance.

**Comment by pgawlowicz** [comment]
Hi, currently it is not on our roadmap.
